packages/devices/src/ble/receiveAPDU.ts

```diff
@@ -1,20 +1,15 @@
-// @flow
-
 import { TransportError, DisconnectedDevice } from "@ledgerhq/errors";
 import { Observable } from "rxjs";
 import { log } from "@ledgerhq/logs";
-
 const TagId = 0x05;
-
 // operator that transform the input raw stream into one apdu response and finishes
 export const receiveAPDU = (
   rawStream: Observable<Buffer>
 ): Observable<Buffer> =>
-  Observable.create((o) => {
+  new Observable((o) => {
```

Comment:
why is it necessary?

Reply:
types say it's deprecated, also, this is just a PoC for now, nothing is settled yet